04_numeronym/madhavansu/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+func numeronyms(vals ...string) []string {
+	strcopy := make([]string, len(vals))
+	for k, val := range vals {
+		val := []rune(strings.TrimSpace(val))
```

Comment:
Maybe use `strings.Trim(val, "\n\t\ ")`. No need to convert to `[]rune` I believe.

Reply:
Updated.